src/main/java/com/cedrus/justin_taylor/kafkademo/Controller/PingPongController.java

```diff
@@ -0,0 +1,42 @@
+package com.cedrus.justin_taylor.kafkademo.Controller;
+
+import com.cedrus.justin_taylor.kafkademo.Config.BallConfig;
+import com.cedrus.justin_taylor.kafkademo.Config.TopicConfig;
+import com.cedrus.justin_taylor.kafkademo.Services.ProducerService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+
+
+@RestController
+public class PingPongController {
+
+    @Autowired
```

Comment:
Please use constructor injection for all injected fields.  It is best to not use different types of injection.